src/Components/MainTableView/TableContent.tsx

```diff
@@ -527,7 +528,8 @@ class TableContent extends React.Component<{token: string, selectedSchemaName: s
                   <input type="checkbox" 
                         // disable multiple check for insert mode as well until multiple insert is supported.
                          disabled={Object.entries(this.state.selectedTableEntries).length > 0 && (this.state.currentSelectedTableActionMenu === TableActionType.DELETE || this.state.currentSelectedTableActionMenu === TableActionType.UPDATE || this.state.currentSelectedTableActionMenu === TableActionType.INSERT) && !this.checkSelection(entry)} 
-                         onChange={(event) => this.handleCheckedEntry(event, entry)} />
+                         onChange={(event) => this.handleCheckedEntry(event, entry)} 
+                         checked={this.checkSelection(entry)}/>
```

Comment:
I think this can be rewritten as a datastorage class to figure out which one has been checked or not. Perhaps file an issue for a possiable rewrite in the future

Reply:
issue filed #61